### ISSUE TITLE
Crashing

### ISSUE THREAD

**Issue by rmccrystal** [davidmann4/ogar-feeder-bot#335]
After about 20 seconds of running bots, I get the error TypeError: Cannot assign to read only property 'type' of reserved fields must be empty

https://gyazo.com/c7ac076fd69195c2bee5d219a3afe3f6

Also, I have 200 working checked proxies, and I only get about 20 bots. How can I fix this?

**Comment by SrNicolas** [comment]
node_modules>agario-client>agario-client.js>search "throw e"

make them ALL like this
//throw err

**Comment by hassanila** [comment]
@SrNicolas That's the worst way to fix the problem, you're just shutting up the problem but not fixing it.

**Comment by Petterholm** [comment]
Bots work?

Sendt fra min iPhone

**Comment by hassanila** [comment]
YES :)

**Comment by Petterholm** [comment]
Does it only work in spectate?

Sendt fra min iPhone

**Comment by ItzLevvie** [comment]
Yes, https://github.com/davidmann4/ogar-feeder-bot/pull/333 {Only works on Party mode.}

**Comment by hassanila** [comment]
WORKS WITH MOUSE

**Comment by Petterholm** [comment]
https://github.com/SrNicolas/agario-feeder-bot

Sendt fra min iPhone
